src/elements/FormParametersTable/FormParametersRow/FormParametersRow.js

```diff
@@ -378,7 +386,13 @@ const FormParametersRow = ({
                     {fieldData.data.type}
                   </Tooltip>
                 </div>
-                <div className="form-table__cell form-table__cell_3">
+                <div
+                  className={classnames(
+                    'form-table__cell',
+                    'form-table__cell_3',
+                    fieldData.isPredefined && 'disabled'
+                  )}
+                >
```

Comment:
Why do you disable value field for predefined parameters

Reply:
@mavdryk the value has a disabled color (light gray) when it's not in edit mode, similar to the Type value color